x-pack/plugins/cloud_integrations/cloud_full_story/server/config.ts

```diff
@@ -18,7 +18,7 @@ const configSchema = schema.object({
     schema.maybe(schema.string())
   ),
   eventTypesAllowlist: schema.arrayOf(schema.string(), {
-    defaultValue: ['Loaded Kibana'],
+    defaultValue: ['Loaded Kibana', 'Hosts View Query Submitted', 'Host Entry Clicked'],
```

Comment:
Curious to see how this will look like in the future, if we start firing more custom events.

Reply:
Adding @afharo as a reviewer, as I'm not sure how this option was intended to be used.

Reply:
Thanks for checking it, this is actually the biggest doubt I want to check since was not clear if this is the source of truth for the whitelisted events, or if they should be set with a different approach (working locally I set them as config in the `kibana.yml` at first attempt, then moved them here)

Reply:
AFAIK, this is the current source of truth (cc @elastic/platform-analytics to keep me honest here). 

FWIW, I don't expect this list to be too large: [FullStory limits](https://help.fullstory.com/hc/en-us/articles/360020623234#property-rate-limiting) are very aggressive: in a session, we can only send 30 custom events per minute (10 per second in a burst). Any further events within that period will be rejected.

If we really need an event in FS (on top of our in-house analytics tool), we can send it. But, IMO, it's worth having this centralized place and having Platform Analytics review the changes.

Having said that, should we start adding comments to each event name about the frequency in which they are sent?

```suggestion
    defaultValue: [
      'Loaded Kibana', // Sent once per page refresh (potentially, once per session)
      'Hosts View Query Submitted', // Worst-case scenario 1 every 2 seconds
      'Host Entry Clicked', // Worst-case scenario once per second - AT RISK
    ],
```